doc/releases/changelog-0.35.0.md

```diff
@@ -4,25 +4,156 @@
+    `qml.from_qiskit`. Consider this simple Qiskit circuit:
+
+    ```python
+    import pennylane as qml
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.rx(0.785, 0)
+    qc.ry(1.57, 1)
+    ```
+    
+    We can convert it into a PennyLane QNode in just a few lines and then add some `measurements`:
+
+    ```pycon
+    >>> dev = qml.device("default.qubit")
+    >>> measurements = [qml.expval(qml.Z(0) @ qml.Z(1))]
+    >>> qfunc = qml.from_qiskit(qc, measurements=measurements)
+    >>> qnode = qml.QNode(qfunc, dev)
+    >>> qnode()
+    [tensor(0.00056331, requires_grad=True)]
+    ```
+
+  * Quantum circuits that already contain Qiskit-side measurements can be faithfully converted with
+    `qml.from_qiskit`. Consider this example Qiskit circuit:
+
+    ```python
+    qc = QuantumCircuit(3, 2)  # Teleportation
+  
+    qc.rx(0.9, 0)  # Prepare input state on qubit 0
+  
+    qc.h(1)  # Prepare Bell state on qubits 1 and 2
+    qc.cx(1, 2)
+  
+    qc.cx(0, 1)  # Perform teleportation
+    qc.h(0)
+    qc.measure(0, 0)
+    qc.measure(1, 1)
+  
+    with qc.if_test((1, 1)):  # Perform first conditional
+        qc.x(2)
+    ```
+    
+    This circuit can be converted into PennyLane with the Qiskit measurements still accessible. For example, we can 
+    use those results as inputs to a mid-circuit measurement in PennyLane:
+
+    ```python
+    @qml.qnode(dev)
+    def teleport():
+        m0, m1 = qml.from_qiskit(qc)()
+        qml.cond(m0, qml.Z)(2)
+        return qml.density_matrix(2)
+    ```
+    
+  * It is now more intuitive to handle parametrized Qiskit circuits. Consider the following circuit:
 
-* A Qiskit `SparsePauliOp` can be converted into a PennyLane `Operator` using `qml.from_qiskit_op`.
+    ```python
+    from qiskit.circuit import Parameter
+
+    angle0 = Parameter("x")
+    angle1 = Parameter("y")
+
+    qc = QuantumCircuit(2, 2)
+    qc.rx(angle0, 0)
+    qc.ry(angle1, 1)
+    qc.cx(1, 0)
+    ```
+    
+    We can convert this circuit into a QNode with two arguments, corresponding to `x` and `y`:
+
+    ```python
+    measurements = qml.expval(qml.PauliZ(0))
+    qfunc = qml.from_qiskit(qc, measurements)
+    qnode = qml.QNode(qfunc, dev)
+    ```
+    
+    The QNode can be evaluated and differentiated:
+
+    ```pycon
+    >>> x, y = qml.numpy.array([0.4, 0.5], requires_grad=True)
+    >>> qnode(x, y)
+    tensor(0.80830707, requires_grad=True)
+    >>> qml.grad(qnode)(x, y)
+    (tensor(-0.34174675, requires_grad=True),
+     tensor(-0.44158016, requires_grad=True))
+    ```
+
+    This shows how easy it is to make a Qiskit circuit differentiable with PennyLane.
+
+  * The `qml.from_qiskit` functionality is compatible with both Qiskit
+    [1.0](https://docs.quantum.ibm.com/api/qiskit/release-notes/1.0) and earlier versions.
+
+* In addition to circuits, it is also possible to convert operators from Qiskit to PennyLane with a new function called
```

Comment:
I tried this and it doesn't work. Is the feature _only_ for SparsePauliOp?

```
from qiskit.quantum_info import Pauli

qiskit_op = Pauli("XYZ")
pl_op = qml.from_qiskit_op(qiskit_op)
```

```
ValueError: The operator XYZ is not a valid Qiskit SparsePauliOp.
```

Reply:
Yes, we opted to focus on `SparsePauliOp`, because it seems to be what Qiskit are recommending going forward, e.g., in the [`opflow` migration guide](https://docs.quantum.ibm.com/api/migration-guides/qiskit-opflow-module) or their [`Estimator` guide](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/dev/qiskit_ibm_runtime.Estimator), they steer people towards `SparsePauliOp`. Note that you can easily convert a `Pauli` into a `SparsePauliOp`, which is what we could recommend if a user asks this.